setup.py

```diff
@@ -85,9 +85,7 @@ def read(filename):
             "externals=pymorize.externals:externals",
         ],
     },
-    package_data={
-        "data.dimensionless_mappings": ["data/dimensionless_mappings.yaml"]
-    },
+    package_data={},
```

Comment:
I don't get it... doesn't this remove any packaged data entirely?

Reply:
That was an old commit. It's outdated.